Microsoft.Research/RegressionTest/ClousotTests/Options.cs

```diff
@@ -4,45 +4,46 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using ClousotTests;
 
 namespace Tests
 {
+    [Flags]
+    public enum TestRuns
+    {
+        None = 0,
+        V40AgainstV35Contracts = 1,
+        V40 = 2
+    }
+
     public class Options
     {
         private const string RelativeRoot = @"..\..\..\..\..\";
         internal const string TestHarnessDirectory = @"Microsoft.Research\RegressionTest\ClousotTestHarness\bin\debug";
-        private static readonly string RootDirectory;
+        protected static readonly string RootDirectory = Path.GetFullPath(RelativeRoot);
 
-        static Options()
-        {
-            RootDirectory = Path.GetFullPath(RelativeRoot);
-        }
+        private readonly string _outDirectory;
```

Comment:
You're right, I'll do a clean up in the next "iteration". 
Should we clean up project by project, as we work on it? Fixing all projects in one step will probably never happen...